File: domain/src/main/java/com/example/entity/Rating.java
package com.example.entity;

import lombok.Getter;
import lombok.RequiredArgsConstructor;

@Getter
@RequiredArgsConstructor
public enum Rating {
    ALL("전체 관람가"),
    TWELVE("12세 관람가"),
    FIFTEEN("15세 관람가"),
    ADULT("청소년 관람 불가");

    private final String description;
}

File: movie/movie-core/src/main/kotlin/com/example/moviecore/domain/movie/model/Rating.kt
package com.example.moviecore.domain.movie.model

enum class Rating(
    val description: String
) {
    ALL("전체 관람가"),
    TWELVE("12세 이상 관람가"),
    FIFTEEN("15세 이상 관람가"),
    ADULT("청소년 관람불가"),
    RESTRICTED("제한상영가")
}